### DISCUSSION TITLE
GraphQL support

### DISCUSSION THREAD

**Discussion by tristan1027** [paradigmxyz/reth#12127] [Help]
Hi team
I'm not a native Rust developer and previously used execution- client is Geth.
I didn't find any introduction to GraphQL in the feature list of RETH. Is it means that RETH does not support GraphQl? will it support GraphQl in subsequent plans?

I'd really like to try upgrading EL to RETH but my program runs partially dependent on the use of GraphQl.
 The usage scenario is: Find the value of multiple storageKeys under the specified Account account under the specified blockNumber. I would like to ask if there is any feature on RETH that can match the above scenario?

Looking forward to your reply. 
Thanks

**Comment by onbjerg** [comment]
Hi Tristan,

We don't have GraphQL support at the moment, and also no current plans to support it. If you want to see the feature added and gauge interest, please open up an issue instead of a discussion:)

**Reply by tristan1027** [reply]
thanks for reply ^-^
